fiftyone/utils/eval/coco.py

```diff
@@ -22,11 +32,23 @@ class COCOEvaluationConfig(DetectionEvaluationConfig):
         classwise (None): whether to only match objects with the same class
             label (True) or allow matches between classes (False)
         iscrowd ("iscrowd"): the name of the crowd attribute
+        compute_mAP (False): whether to perform the necessary computations so
+            that mAP and PR curves can be generated
+        iou_threshs ([0.5::0.05::0.95]): a list of IoU thresholds to use when
+            computing mAP and PR curves
     """
 
-    def __init__(self, iscrowd="iscrowd", **kwargs):
+    def __init__(
+        self, iscrowd="iscrowd", compute_mAP=False, iou_threshs=None, **kwargs
+    ):
         super().__init__(**kwargs)
+
+        if compute_mAP and iou_threshs is None:
```

Comment:
I tweaked this so that `iou_threshs` is only populated if `compute_mAP == True`. Otherwise, the `get_evaluation_info()` would store a list of `iou_threshs` even when no sweep was performed, which is misleading